packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1128,6 +1276,51 @@ export class Table extends TableBase {
   protected get hasIndex(): boolean {
     return this.globalSecondaryIndexes.length + this.localSecondaryIndexes.length > 0;
   }
+
+  /**
+   * Set up key properties and return the Table encryption property from the
+   * user's configuration.
+   */
+  private parseEncryption(props: TableProps): {sseSpecification: CfnTable.SSESpecificationProperty, encryptionKey?: kms.IKey} {
+    let encryptionType = props.encryption;
+
+    if (encryptionType !== undefined && props.serverSideEncryption) {
+      throw new Error('Both encryption and serverSideEncryption is specified, only either field can be set, not both');
+    }
+
+    if (encryptionType === undefined) {
+      encryptionType = props.encryptionKey ? TableEncryption.CUSTOMER_MANAGED : TableEncryption.DEFAULT;
+    }
+
+    if (props.serverSideEncryption && props.encryptionKey) {
+      throw new Error('encryptionKey cannot be specified for serverSideEncryption, use encryption instead');
+    }
+
+    if (encryptionType !== TableEncryption.CUSTOMER_MANAGED && props.encryptionKey) {
+      throw new Error('encryptionKey is specified, so encryption must be set to Customer_Managed');
+    }
+
+    if (encryptionType === TableEncryption.CUSTOMER_MANAGED && props.replicationRegions) {
+      throw new Error('Global tables cannot be encrypted with customer managed CMK');
+    }
+
+    if (encryptionType === TableEncryption.CUSTOMER_MANAGED) {
+      const encryptionKey = props.encryptionKey || new kms.Key(this, 'Key', {
+        description: `Created by ${this.node.path}`,
+        enableKeyRotation: true,
+      });
+
+      return { sseSpecification: {sseEnabled: true, kmsMasterKeyId: encryptionKey.keyArn, sseType: 'KMS'}, encryptionKey };
+    }
+
+    if (encryptionType === TableEncryption.AWS_MANAGED || props.serverSideEncryption) {
+      return { sseSpecification: {sseEnabled: true, sseType: 'KMS'} };
+    }
+    if (encryptionType === TableEncryption.DEFAULT) {
+      return { sseSpecification: {sseEnabled: false} };
```

Comment:
(Yeah I'm obsessive about consistent spacing :D)

```suggestion
      return { sseSpecification: { sseEnabled: false } };
```